packages/attendance/src/services/classServiceRegistry.js

```diff
@@ -1,4 +1,4 @@
-import { get, post } from "@shiksha/common-lib";
+import { get, post, update as coreUpdate } from "@shiksha/common-lib";
```

Comment:
I think the file should be named `classRegistryService`

Reply:
Done